src/javascript/app/pages/callback/callback.jsx

```diff
@@ -113,7 +113,7 @@ const CallbackContainer = () => {
             }
             getElementById('loading_link').setAttribute('href', redirect_url);
             
-            window.location.href = redirect_url; // need to redirect not using pjax
+            window.location.replace(redirect_url); // need to redirect not using pjax
```

Comment:
I think `.replace` doesn't redirect user to the url location, you should `.replace` and redirect

Reply:
it does redirect right? I guess I did not follow your comment